src/common/completion_docs.ts

```diff
@@ -342,25 +319,25 @@ export const COMPLETION_DOCS: {
   model_property: {
     source: MODEL_SOURCE_DOC,
     query: MODEL_QUERY_DOC,
-    sql: MODEL_SQL_DOC,
+    run: MODEL_RUN_DOC,
   },
   query_property: {
     group_by: QUERY_GROUP_BY_DOC,
     order_by: QUERY_ORDER_BY_DOC,
-    project: QUERY_PROJECT_DOC,
+    select: QUERY_SELECT_DOC,
     index: QUERY_INDEX_DOC,
     aggregate: QUERY_AGGREGATE_DOC,
     top: QUERY_TOP_DOC,
     limit: QUERY_LIMIT_DOC,
     where: QUERY_WHERE_DOC,
     having: QUERY_HAVING_DOC,
     nest: QUERY_NEST_DOC,
-    declare: QUERY_DECLARE_DOC,
+    calculate: QUERY_CALCULATE_DOC,
   },
   explore_property: {
     dimension: SOURCE_DIMENSION_DOC,
     measure: SOURCE_MEASURE_DOC,
-    query: SOURCE_QUERY_DOC,
+    view: SOURCE_VIEW_DOC,
```

Comment:
These values have a companion tables in the [malloy](https://github.com/malloydata/malloy/blob/main/packages/malloy/src/lang/parse-tree-walkers/document-completion-walker.ts#L35-L65)
I had a PR up for `query` to `view` but I'm realizing they're quite out of date, or at least not at m4 yet? Unless there's a branch.

Reply:
So AFAIK property completions are just completely broken right now, so I'm not worrying about fixing them at the moment. And I believe that list you're talking about is just for property completions, not property hovers.